src/main/java/core/basesyntax/repoter/ReportGenerator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.repoter;
```

Comment:
The package name 'repoter' seems to be a typo. It should be 'reporter' to correctly represent its functionality.